### PR TITLE
Create the course description page

### PR DESCRIPTION
**Pull request by tavy88**
### Context

Add the following formatting rules:

Price:
- For “0.00”, show “Free”
- for any other number, show the actual price
- if nil, don’t show row

Description:
- if less than 15 chars hide
- display otherwise

Start Date:
- format to %d %b %Y

End Date:
- removed for now due to too much data inconsistency

Website url:
- Courses - Booking url OR
- Courses - Url OR
- Provider - Url OR
- Venue url

Also fixing an accessibility issue on the main courses
listing page.

### Ticket

https://dfedigital.atlassian.net/browse/GET-1023

### CODE REVIEW

File: app/decorators/csv/opportunity_decorator.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        .concat('<br>')
+        .concat(full_region)
+        .concat('<br>')
+        .concat(postcode)
+        .html_safe
+    end
+
+    def course_start_date
+      return nil unless opportunity_start_dates.any?
+      return 'Different start dates' if opportunity_start_dates.size > 1
```
Comment (tavy88) [inline app/decorators/csv/opportunity_decorator.rb]: To validate with Lois the copy here.

File: config/locales/en.yml [hunk 2]
```diff
@@ -16,6 +16,8 @@ en-GB:
     job_profile: Job profile details
     your_skills: Your skills
     training_courses_near_you: Find and apply to training courses near you
+    training_courses_near_you: Training courses near you
```
Comment (tavy88) [inline config/locales/en.yml]: This should be `training_courses_near_me`

File: app/views/courses/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,91 @@
+<% page_title :courses_index %>
```
Comment (cwrw) [inline app/views/courses/show.html.erb]: change to `course_detail`

File: spec/routing/courses_spec.rb [hunk 4]
```diff
@@ -10,4 +10,38 @@
...
+    end
+  end
+
+  context 'when csv_courses flag is ON' do
+    before do
+      enable_feature!(:csv_courses)
+    end
+
+    it 'does not route to courses#show' do
+      course = create(:csv_course)
```
Comment (cwrw) [inline spec/routing/courses_spec.rb]: remove extra factories

**Pull request merged**